profile/README.md

```diff
@@ -10,3 +10,45 @@
 🍿 Fun facts - what does your team eat for breakfast?
 🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
 -->
+
+As “Big Data” cancer researchers, we both apply existing techniques and continuously work to improve them and create new methods to create clinically useful diagnostics that personalize therapy.
+
+Much of our work to date has focused on prostate cancer, but we are increasingly applying the same techniques to cancers of the kidney, bladder and other sites, and discovering fascinating commonalities across them.
+
+***Want to learn more?***
+<br>Go to the [Cancer Data Sciences page](https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab) where you can
+read about our recent [research areas](https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab/research-areas) and some of our recent [publications](https://www.uclahealth.org/departments/urology/iuo/research/faculty-labs/dr-paul-boutros-lab/publications).
+
+--- 
+
+## :bar_chart: Tools
+From this organization GitHub page, you can find our **R packages, Docker images, and Nextflow pipelines.** 
```

Comment:
We haven't made Nextflow pipelines public yet so maybe focus on R packages for now?